src/main/java/camp/nextstep/edu/rebellion/lotto/domain/LottoAward.java

```diff
@@ -0,0 +1,38 @@
+package camp.nextstep.edu.rebellion.lotto.domain;
+
+import java.util.Arrays;
+
+public enum LottoAward {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
위 처럼 표현하여 가독성을 높일 수 있어요.

Reply:
😮 int 형인데 저렇게 표현이 가능한가요?? 😮